drivers/vulkan/rendering_device_vulkan.cpp

```diff
@@ -1463,7 +1463,7 @@ Error RenderingDeviceVulkan::_buffer_update(Buffer *p_buffer, size_t p_offset, c
 		{
 			VkResult vkerr = vmaMapMemory(allocator, staging_buffer_blocks[staging_buffer_current].allocation, &data_ptr);
 			if (vkerr) {
-				ERR_FAIL_V(ERR_CANT_CREATE);
+				ERR_FAIL_V_MSG(ERR_CANT_CREATE, "vmaMapMemory failed with error " + itos(vkerr) + ".");
```

Comment:
This could be `ERR_FAIL_COND_V_MSG(vkerr, ERR_CANT_CREATE, "...");` (same for all other occurrences where the condition is check before a single `ERR_FAIL_V_MSG()`.)

Reply:
Alright, I'm going to change these to `ERR_FAIL_COND_V_MSG`.

Reply:
Done!